spec/features/hiring_staff_can_publish_a_vacancy_spec.rb

```diff
@@ -119,6 +119,23 @@
         expect(page).to have_content(I18n.t('jobs.current_step', step: 3, total: 6))
         expect(page.current_path).to eq(supporting_documents_school_job_path)
       end
+
+      scenario 'saves draft and redirects to school page when submitted as a draft' do
+        visit new_school_job_path
+
+        fill_in_job_specification_form_fields(vacancy)
+        click_on I18n.t('buttons.save_and_continue')
+
+        fill_in_pay_package_form_fields(vacancy)
+        click_on I18n.t('buttons.save_and_return')
+
+        expect(page.current_path).to eq(jobs_with_type_school_path('draft'))
+        click_on vacancy.job_title
```

Comment:
Can we click on the 'Edit' within the row instead

Reply:
Yes, it converges to the same page via a different sequence of redirects. But I will update the test as that's a better thing to click.